pkgs/applications/science/machine-learning/openbugs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, stdenv
+, fetchurl
+, gcc_multi
```

Comment:
```suggestion
```

Reply:
This package is mainly 64-bit, but previously @nevivurn told me a way to support also 32-bit architecture. In 64 bits the gcc_multi is needed because this package depends on a library of gnu/stub-32.h, but in 32-bit it doesn't. 
Probably, this library is to be able to compile the package for 64 bit architecture, otherwise it does not compile.

I have tried in building and running the program in both architectures and they work without a problem.

I don't know if pkgsi686Linux is able to package for 64 bit (I am new :)).

Reply:
The package _can't_ be "mainly 64-bit", because the package

```make
LDFLAGS = -L ../lib -Wl,-rpath,\$$ORIGIN/../lib -m32 
```

x86_64 can run 32-bit binaries, so if you've used this package before, that's almost certainly what's been happening.

To check I'm correct about this, I built the package in an x86_64 Fedora container, and it produced a 32-bit binary:

```
src/OpenBUGSCli: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux.so.2, BuildID[sha1]=57db2dd73a4471086ffcf4e3208776d5013b3fc0, for GNU/Linux 3.2.0, not stripped
```

Reply:
Oh, interesting. Sorry for the confusion. After doing some research, you are correct.  I will fix it. And thank you, I am still learning to package software. I have a lot to learn!